requirements/test.txt

```diff
@@ -6,3 +6,4 @@ coverage
 python-coveralls
 mock
 flake8
+colorlog
```

Comment:
I'm not familiar with `colorlog`, i googled it but i'm still confused. Does add colors to `pytest` or `flake8` logs ? i checked the `travis-ci` builds before and after but i didn't notice any difference 🤔

Reply:
I'm removing this. I jumped the gun with adding this based on local setup